assets/app/scripts/filters/resources.js

```diff
@@ -231,6 +231,17 @@ angular.module('openshiftConsole')
       }
     };
   })
+  .filter('stripTag', function() {
```

Comment:
i feel like this file is getting too big, we're over 1000 lines already.  don't need to fix it here, but we should probably split it up soon